components/filter/index.js

```diff
@@ -1,19 +1,23 @@
 import styles from './filter.module.css';
+import Image from 'next/image';
 
 const Filter = (props) => {
   const { changeTransactions } = props;
   return (
     <div className={styles.filter}>
-      <img
-        className="icon"
-        src="https://www.flaticon.com/svg/static/icons/svg/3126/3126539.svg"
-        alt="Filter icon"
-        onClick={() => {
-          document
-            .getElementById('filterContent')
-            .classList.toggle(styles.showList);
-        }}
-      />
+      <div className="icon">
+        <Image
+          src="/assets/filter-icon.svg"
```

Comment:
Somewhere it is path.join and some where direct string. can we have a consistent style for same.

Reply:
yes.